spring-integration-zip/src/test/java/org/springframework/integration/zip/transformer/UnZipTransformerTests.java

```diff
@@ -226,8 +245,7 @@ public void unzipMultipleFilesAsInputStreamWithExpectSingleResultTrue() throws I
 
 		try {
 			unZipTransformer.transform(message);
-		}
-		catch (MessagingException e) {
+		} catch (MessagingException e) {
```

Comment:
We indeed require to have `catch` on a new line, as well as `else`, `finally` etc.